CMakeLists.txt

```diff
@@ -47,11 +47,12 @@ option(WITH_TITAN_TOOLS "Build with tools." ON)
 option(TRAVIS "Building in Travis." OFF)
 option(CODE_COVERAGE "Generate code coverage report." OFF)
 
-if (CMAKE_BUILD_TYPE STREQUAL "Release")
+if (CMAKE_COMPILER_IS_GNUCXX)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-array-bounds")
```

Comment:
I'm worried this may prevent founding errors early. Is it possible to enable the check in titan tests while disabling it for rust-rocksdb?

Reply:
Let's merge this cherry-pick first. I'll see if I can only disable the check for the specific line of code.